worlds/jakanddaxter/Regions.py

```diff
@@ -170,41 +176,19 @@ def verify_connector_level_accessibility(multiworld: MultiWorld, options: JakAnd
         option = thresholds[k]["option"]
         required_items = thresholds[k]["required_items"]
 
-        # Given our current state (starting with 0 Power Cells), determine if there are enough
+        # Given our current fake_state (starting with 0 Power Cells), determine if there are enough
         # Locations to fill with the number of Power Cells needed for the next threshold.
-        locations_available = multiworld.get_reachable_locations(state, player)
+        locations_available = multiworld.get_reachable_locations(fake_state)
```

Comment:
Oh we don't even use that function anymore. I think I kept it around because it still seemed useful, but what do you think? Keep it and use it? Don't use it, don't keep it?

Reply:
Since it's not used, it should either be commented out (with an explanatory comment) or just removed

Reply:
I mean, it's in the git history. If we need it again, we can grab it. I also have a misc folder of useful scripts, functions, etc. that I can save for quick reference.